ee/clickhouse/queries/event_query.py

```diff
@@ -1,39 +1,21 @@
-from abc import ABCMeta, abstractmethod
-from typing import Any, Dict, List, Optional, Tuple, Union
+from typing import Dict, List, Optional, Tuple, Union
 
 from ee.clickhouse.materialized_columns.columns import ColumnName
-from ee.clickhouse.models.cohort import is_precalculated_query
-from ee.clickhouse.models.property import parse_prop_grouped_clauses
-from ee.clickhouse.models.util import PersonPropertiesMode
-from ee.clickhouse.queries.column_optimizer import ColumnOptimizer
+from ee.clickhouse.queries.column_optimizer import EE_ColumnOptimizer
 from ee.clickhouse.queries.groups_join_query import GroupsJoinQuery
-from ee.clickhouse.queries.person_distinct_id_query import get_team_distinct_ids_query
-from ee.clickhouse.queries.person_query import ClickhousePersonQuery
-from ee.clickhouse.queries.util import parse_timestamps
-from posthog.models import Cohort, Filter, Property
-from posthog.models.filters.mixins.utils import cached_property
+from posthog.models.filters.filter import Filter
 from posthog.models.filters.path_filter import PathFilter
 from posthog.models.filters.retention_filter import RetentionFilter
 from posthog.models.filters.session_recordings_filter import SessionRecordingsFilter
 from posthog.models.filters.stickiness_filter import StickinessFilter
-from posthog.models.property import PropertyGroup, PropertyName
+from posthog.models.property import PropertyName
 from posthog.models.team import Team
+from posthog.queries.event_query import EventQuery
 
 
-class ClickhouseEventQuery(metaclass=ABCMeta):
-    DISTINCT_ID_TABLE_ALIAS = "pdi"
-    PERSON_TABLE_ALIAS = "person"
-    EVENT_TABLE_ALIAS = "e"
+class EE_EventQuery(EventQuery):
```

Comment:
.. and same for everything following this pattern?

... or is this more like a new pattern, where EE related code for that specific class is named like so?

Reply:
haha yeah let's do that